src/formatDate.js

```diff
@@ -8,7 +8,32 @@
  * @returns {string}
  */
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const tempArr = date.split(fromFormat[3]);
```

Comment:
The code assumes that the delimiter is always at index 3 of the `fromFormat` array. This might not be the case for all date formats. Consider making the delimiter dynamic or configurable.